### ISSUE TITLE
this doesnt appear to handle throughput / throttling...

### ISSUE THREAD

**Issue by ezeeetm** [techgaun/dynamodb-copy-table#4]
given a source table of x read throughput and a dest table of <x write throughput, this utility will result in throttled writes, which means the throttled data will not be copied to dest.

The converse of this could also exist: a throttled read resulting in nothing returned, which then never gets written.

Am I observing this correctly, or am I missing something?

If so, I may be able to do a PR For something to fix it.  Let me know.

**Comment by techgaun** [comment]
@ezeeetm you are correct. this is very raw script and does not handle throughputs. It would be great if you could send a PR for the fix. I might not be able to spend time at least for a month because of work.
